doc/source/configuration/monitoring.rst

```diff
@@ -1,4 +1,8 @@
-========================
+===========
+Monitoring
+===========
+
+
 Monitoring Configuration
 ========================
 
```

Comment:
Let's provide a bit more information about what's included in the stock config. Which services are enabled? What custom config is provided (probably don't need to list dashboards or rules files, but at least note that they are present).

Reply:
Added